CR-757 Add FormType, QuestionnaireType

MartinHumphrey opened the pull request:
# Motivation and Context
Simple change to add FormType and QuestionnaireType to the response for a request to the mock RM Case service endpoint for the url /cases/{caseId}/qid to get a new questionnaire Id for telephone capture.

# What has changed
Addition of required fields to CaseServiceMockStub for appropriate response. Can be tested running mock case service and using Postman to issue the GET request:
//localhost:8161/cases/03f58cb5-9af4-4d40-9d60-c124c5bddf09/qid

MartinHumphrey committed:
CR-757 Add FormType, QuestionnaireType

Add new fields to mock endpoint to generate a new questionnaire Id for a case.

PeterBochel commented on pom.xml:
```diff
@@ -164,7 +164,7 @@
             <groupId>uk.gov.ons.ctp.integration.common</groupId>
             <artifactId>census-int-case-api-client</artifactId>
-            <version>0.0.7</version>
+            <version>0.0.8-SNAPSHOT</version>
```
It shouldn't be using a snapshot release.

MartinHumphrey replied:
Agreed. It is waiting approval of this [case client api pull request](https://github.com/ONSdigital/census-int-case-api-client/pull/7). It is a facet of our continuous integration build I believe. The version will move to 0.0.8 when the case client api pull request is approved and I will be able to update this dependency in the POM here by removing the "-SNAPSHOT". 
Maybe a bit premature in raising this pull request, the other should be merged first then I can change this to the final version before this is merged. One simple little change I didn't wait for but maybe I shouldn't have raised this pull request quite yet.

MartinHumphrey replied:
Done. Dependency updated to final 0.0.8 version now associated pull request approved.

MartinHumphrey committed:
CR-757 Update case-api-client dependency to final version

Remove Snapshot dependency now associated case-api-client pull request approve. Remove space at end of line in  mock stub code.
